sklearn/decomposition/tests/test_incremental_pca.py

```diff
@@ -188,7 +189,7 @@ def test_incremental_pca_batch_rank():
         all_components.append(ipca.components_)
 
     for i, j in zip(all_components[:-1], all_components[1:]):
```

Comment:
Also maybe let's rename these to `components_i`, `components_j` as suggested in https://github.com/scikit-learn/scikit-learn/pull/12379#discussion_r229483597 ?

Reply:
@rth many thanks! it's done.